ebml/EbmlTypes.h

```diff
@@ -33,33 +33,30 @@
 #ifndef LIBEBML_TYPES_H
 #define LIBEBML_TYPES_H
 
+#include <cstdint>
+
 #include "ebml/c/libebml_t.h"
 #include "ebml/EbmlConfig.h"
 #include "EbmlEndian.h" // binary needs to be defined
 
 START_LIBEBML_NAMESPACE
 
-typedef wchar_t utf16;
```

Comment:
For backward compatibility you need to keep the old typedefs to the newer values.